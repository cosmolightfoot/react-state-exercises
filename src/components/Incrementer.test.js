import React from 'react';
import { shallow } from 'enzyme';
import Incrementer from './Incrementer.js';

describe('Incrementer tests', () => {
  it('renders the component', () => {
    const wrapper = shallow(<Incrementer />);
    expect(wrapper).toMatchSnapshot();
  });
  it('increments count on a button click', () => {
    const wrapper = shallow(<Incrementer />);
    wrapper.find('button').at(0).simulate('click');
    expect(wrapper.state('count')).toEqual(1)
  });
});
