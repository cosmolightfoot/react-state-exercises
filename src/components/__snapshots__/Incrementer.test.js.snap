// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Incrementer tests renders the component 1`] = `
<Fragment>
  <p>
    Times Clicked: 
    0
  </p>
  <button
    onClick={[Function]}
  >
    Click Me!
  </button>
</Fragment>
`;
